### DISCUSSION TITLE
Polkadot support

### DISCUSSION THREAD

**Discussion by paulvi** [web3j/web3j#1500] [Ideas]
## Feature description_
Polkadot https://polkadot.network/ is other blockchain 
(see 2019 comparison https://www.purestake.com/blog/ethereum-vs-polkadot/)

However, I bet some developer would like to see polkadot client library to be exactly or like web3j, maybe by we3 labs.
There is https://github.com/polkadot-java/api implementation that is abandoned  https://github.com/polkadot-java/api/issues/34 and have different idea and tech than web3j.

This issues is raised to ask, if you ever considered 
Or it is definitely  out of scope, so will not work on it.

**Comment by mohamedelshami** [comment]
Hi @paulvi, this is certainly something we can consider support for in Web3j but it's not something in the scope in the short term.

**Comment by paulvi** [comment]
OK, having it separately and visible is just exact good handling.

let's see how many people vote here +1 and how much discussion will happen....

**Comment by mohamedelshami** [comment]
Yes, it will be good to get some insight on how the interest from the community. I pinned the discussion.

I haven't done much research on Polkadot client APIs - is (https://github.com/polkadot-java/api) the only Java client library available so far?

**Comment by paulvi** [comment]
Other is https://github.com/emeraldpay/polkaj, both in a (bit) broken state as things move fast

**Comment by jambestwick** [comment]
polkadot is the chain hub , web3j is api frame  support EVM
they are different!!!

**Comment by mohamedelshami** [comment]
Web3j is a client library for any blockchain with Web3 API compatible RPC, it's not limited to Ethereum or EVM.
